src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
+        ColorSupplier suppler = new ColorSupplier();
+        int ballNumber = random.nextInt(100);
+        String ballColor = suppler.getRandomColor();
+        return new Ball(ballColor,ballNumber);
```

Comment:
Magic number '100' is used here. It's better to declare it as a constant field to give it a meaningful name and to make the code more maintainable. For example, you could name it `MAX_BALL_NUMBER`.